test/phpunit.xml.dist

```diff
@@ -2,9 +2,8 @@
 <phpunit bootstrap="bootstrap.php">
   <testsuites>
     <testsuite name="General">
-      <file>./smtpapi.php</file>
-      <file>./smtpapi/Header.php</file>
-      <file>./RepoFilesExists.php</file>
+      <file>./Smtpapi.php</file>
+      <file>./Smtpapi/Header.php</file>
```

Comment:
I don't think it's the proper way to define `tests` files with `file` tag.

How about using the `directory` tag to set the `tests` files?

Reply:
I'd agree, I was just working off what was already in the file :)

Reply:
Although if I make that change right now, I would have to add an explicit exception for the RepoFilesExists test that is broken. Maybe this is okay for now, and can be revisited once more tests are added?